app/src/main/res/values/strings.xml

```diff
@@ -2,16 +2,16 @@
 <resources>
 
     <!-- Applications list -->
-    <string name="info_applications_list_refreshed">Applications list refreshed</string>

```

Comment:
Plural error

Reply:
Hum, I thought both forms were tolerated for "application".
For "app" however, there is a plural form "apps" (as you used in line 10)
=> Could it rather be "Apps list refreshed"?

Reply:
The list of apps would be the "app list".
There is no pluralization for attributive nouns. It isn't more of the actual thing, but rather the concept, in which many things fit.
For example a "mail server" isn't "mails server" even though they hold many e-mails.
Car garage
Car garages (many garages suited for cars)
Cars garage (the cars are doing the act of garaging)

Reply:
Ok, thank you for the explanation!
In French there would be a plural in this case (for example, the mail server would be "un serveur de mails") which is why a plural seemed more natural to me, I didn't know it was a mistake in English.
=> Correction OK for me